examples/high-availability/etcd.manifest

```diff
@@ -0,0 +1,97 @@
+{
+"apiVersion": "v1beta3",
+"kind": "Pod",
+"metadata": {"name":"etcd-server"},
+"spec":{
+    "hostNetwork": true,
+    "containers":[
+    {
+    "name": "etcd-container",
+    "image": "quay.io/coreos/etcd",
+    "command": [
+          "/etcd","--name","NODE_NAME",
+	      "--initial-advertise-peer-urls",  "http://NODE_IP:2380",
+	      "--listen-peer-urls",     	"http://NODE_IP:2380",
+	      "--advertise-client-urls",    	"http://NODE_IP:2379",
+	      "-initial-cluster", "kube0.ha=http://192.168.4.100:2380",
+	      "--listen-client-urls",   	"http://127.0.0.1:2379,http://NODE_IP:2379",
+          "--data-dir","/var/etcd/data"
+    ],
+    "ports":[
+        {
+            "name": "serverport", "containerPort": 2380, "hostPort": 2380
+        },
+        {
+            "name": "clientport","containerPort": 4001, "hostPort": 4001
+        }
+    ],
+    "volumeMounts": [
+        { "name": "varetcd",
```

Comment:
would be useful to put each mount on a single line, in columns -- as is, it's really long